src/main/resources/themes/vscode_dark.xml

```diff
@@ -1122,6 +1128,11 @@
                 <option name="EFFECT_TYPE" value="5"/>
             </value>
         </option>
+        <option name="XML_CUSTOM_TAG_NAME">
+            <value>
+                <option name="FOREGROUND" value="40c8ae"/>
```

Comment:
To keep it consistent please update this to: 
```suggestion
                <option name="FOREGROUND" value="4ec9b0"/>
```

Reply:
Done ✔️